testing/conformance/Cargo.toml

```diff
@@ -36,6 +35,7 @@ regex = { version = "1.0" }
 ittapi-rs = { version = "0.3.0", optional = true }
 actors-v10 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "next" }
 libipld-core = { version = "0.14.0", features = ["serde-codec"] }
+multihash = { version = "=0.16.2", default-features = false }
```

Comment:
Since this code is disabled in the top-level workspace Cargo.toml, changing this shouldn't be necessary. If the dep can actually be a level down, better to submit that upstream.